Src/IronPython/Runtime/Operations/IntOps.cs

```diff
@@ -174,12 +174,12 @@ public static partial class Int32Ops {
         }
 
         [StaticExtensionMethod]
-        public static object __new__(CodeContext/*!*/ context, PythonType cls, IList<byte> s) {
+        public static object __new__(CodeContext/*!*/ context, PythonType cls, IList<byte> s, int @base=10) {
```

Comment:
Should this be `[DefaultParameterValue(10)]` instead? Or are we just using `DefaultParameterValue` all over the place because IronPython predates optional arguments? I'm not really sure...